docs/install/_linux.rst

```diff
@@ -37,18 +39,81 @@ The advantages of downloading a ``.deb`` file is the portability: you can copy t
 
        kolibri start
 	
-#. When command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
+#. When the command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
 
 
 Uninstall
----------
+*********
 
 * Open **Software** on Ubuntu and locate the Kolibri. Press **Remove**.
-* Or from command line: ``sudo apt-get remove kolibri``.
+* Or from the command line: ``sudo apt-get remove kolibri``.
+
 
 Upgrade
--------
+*******
 
 When you use the PPA installation method, upgrades to newer versions will be automatic, provided there is internet access available.
 
 To upgrade Kolibri on a Debian device without internet access, bring the updated ``.deb`` file and follow the same steps as in :ref:`lin_deb`.
+
+
+.. _rpi:
+
+Raspberry Pi (Raspbian)
+-----------------------
+
+Kolibri currently doesn't work out of the box for Raspbian Jessie. We are refining our distribution to work out of the box, but you need to follow these two steps in order to install it (tested on Kolibri 0.9):
+
+1. Running ``add-apt-repository`` as shown in the PPA instructions does not work. Instead, run::
+
+      echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list
```

Comment:
@benjaoming I'm currently on a raspberry pi 3 with Raspbian Jessie. I run the command `echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list` but it errors out with `bash: /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list: Permission denied`. I also try to run the command with `sudo`, but it doesn't work. I think the problem here is that the file `/etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list` has never been created. So I'm wondering before running this command, it would be better to create a file first? Do you have any suggestions? Thanks!

Reply:
ah thank you so much! that totally makes sense!

Reply:
@benjaoming I'm also wondering that instead of naming it a `raspberry pi` issue, would it be okay to name it a `debian jessie` issue? I tested on debian jessie days ago and found the same issue there: https://github.com/learningequality/kolibri-installer-debian/issues/30#issuecomment-381781511. My guess is that it's more likely a debian jessie issue?